api/src/pcapi/core/offers/factories.py

```diff
@@ -219,6 +219,10 @@ def _create(
         return super()._create(model_class, *args, **kwargs)
 
 
+class DraftOfferFactory(OfferFactory):
+    isActive = False
+
```

Comment:
Je crois qu'il faut que tu rajoutes cela pour que ta factory soit complète.

```suggestion
class DraftOfferFactory(OfferFactory):
    isActive = False
    validation = OfferValidationStatus.DRAFT

```